RFCs/0019-Disable-Appdomain-While-Running-Tests.md

```diff
@@ -0,0 +1,28 @@
+# 0019 Disable Appdomain while Running Tests
+
+## Summary
+The RFC outlines why
+
+1. Test adapters should honour disable app domain setting
+
+2. Running of tests with disable appdomain is important for resiliency 
+
+## Motivation
+In past we have seen customer hitting issue with Appdomain.Unload. There are two main issues with Appdoamin.Unload call
```

Comment:
Can we give some more detail here please?

Reply:
done